features/zones/zone-cps/egresses/Item.feature

```diff
@@ -31,8 +34,8 @@ Feature: zones / egresses / item
       """
     When I visit the "/zones/zone-cp-1/egresses/item-1/overview" URL
     Then the page title contains "item-1"
-    Then the "$egress-detail-tabs-view" element contains "item-1"
-    Then the "$egress-detail-view" element contains "166.197.238.26:20555"
-    Then the "$egress-detail-view" element contains "Connected: Jul 28, 2020, 4:18 PM"
-    When I click the "$egress-config-tab" element
-    Then the "$egress-config-view" element contains "type: ZoneEgress"
+    Then the "$header" element contains "item-1"
+    Then the "$overview-view" element contains "166.197.238.26:20555"
+    Then the "$subscription" element contains "Jul 28, 2020, 4:18 PM"
+    When I click the "$config-tab" element
+    Then the "$config-view" element contains "type: ZoneEgress"
```

Comment:
This is testing exactly the same as in ingresses, so I made the code the same also, eventually I'd like to move these tests in a separate subscriptions set of tests and see if we can run them in all 4 areas.